angstudy/src/app/services/product-card.service.ts

```diff
@@ -7,7 +7,7 @@ import { IProduct } from '../interface/product';
 export class ProductCardService {
 	constructor(private http: HttpClient) {}
 
-	getProduct$(): Observable<IProduct> {
-		return this.http.get<IProduct>('/product');
+	getProduct$(productId: string): Observable<IProduct> {
+		return this.http.get<IProduct>('/product/'+productId);
```

Comment:
```suggestion
		return this.http.get<IProduct>(`/product/${productId}`);
```
Мне кажется так поприятней

Reply:
Исправил